tests/bluetooth/common/testlib/src/timeout.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
```

Comment:
Please provide a description telling that building this file will add an extra mandatory command line argument to the executable.
And given that the option would be mandatory, but the use is only if one wants for some reason this timeout_get() should this be behind a kconfig?

Reply:
Fair enough, then maybe we should call it `-test_deadline` and make it a convention for bsim tests in Zephyr which use this library.